Library/Homebrew/utils/github.rb

```diff
@@ -2,6 +2,8 @@
 # frozen_string_literal: true
 
 require "uri"
+require "utils/curl"
+require "utils/popen"
```

Comment:
It would be nice to include these only in the methods that need them.

Reply:
I wasn't sure about some of these, so thanks for confirming. I'll create a follow-up PR in a moment.